modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -346,6 +347,178 @@ public void testAddObjectLayout() throws Exception {
 						"than 12"));
 		}
 
+		_objectDefinitionLocalService.deleteObjectDefinition(
```

Comment:
@p-albuquerque, these tests need to be ordered by exception name and exception message. Could you please reorder these tests?